### DISCUSSION TITLE
Revisit Bool Props

### DISCUSSION THREAD

**Discussion by jonathanprozzi** [HausDAO/daohaus-monorepo#109] [Development]
Pulling this in from #105 to test out as a Discussion topic. 

Thanks @jordanlesich for starting this conversation.

# What

Have a meeting where we revisit the bool prop pattern (trying to use boolean props wherever possible), and make changes to the spec if we decide on something different. 

Bonus round: 

Let's also look at anything else that might be a pattern we don't like. 

# Why

Going back and forth on pattern discussion, with hours in response time between each comment, should be avoided. It's what I'm spending the majority of my time on. 

I really do enjoy reviews and feedback, but it's counterproductive if we're not on the same page when it comes to key patterns. 

- [ ] If we do reach some new patterns, make changes to the spec.

**Comment by jonathanprozzi** [comment]
@jordanlesich @alexkeating @skuhlmann @earth2travis 

Let's try this out to see if we like this format for longer form discussions. This allows us to create issues from discussions, so maybe we could start discussion threads and then create issues from them? Open to trying out a new process with this -- I haven't used this feature before.

**Comment by earth2travis** [comment]
How does Radix handle these props?

**Comment by earth2travis** [comment]
@jordanlesich and @alexkeating do y'all have any thoughts on this?

**Comment by jordanlesich** [comment]
Sure. This keeps a record of the discussion in general, so if we're going to occasionally unblock larger pattern/spec related notes, then this seems pretty good.

**Comment by earth2travis** [comment]
Testing notifications

**Comment by skuhlmann** [comment]
TRUE OR FALSE????

**Comment by skuhlmann** [comment]
@jordanlesich @jord
